webapp/src/selectors.js

```diff
@@ -27,6 +27,7 @@ export const getMessage = (state) => {
 export const getIssues = (state) => getPluginState(state).issues;
 export const getInIssues = (state) => getPluginState(state).inIssues;
 export const getOutIssues = (state) => getPluginState(state).outIssues;
```

Comment:
These selectors should probably be doing this now:

```suggestion
export const getIssues = (state) => getAllIssues(state).my;
export const getInIssues = (state) => getAllIssues(state).in;
export const getOutIssues = (state) => getAllIssues(state).out;
```